src/Sylius/Bundle/ShopBundle/Resources/config/routing/partial/cart.yml

```diff
@@ -0,0 +1,17 @@
+# This file is a part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+sylius_shop_partial_cart_item_create:
+    path: /new
+    methods: [GET, POST]
+    defaults:
+        _controller: sylius.controller.cart_item:createAction
+        _sylius:
+            factory:
+                method: createForProductWithId
+                arguments: [$productId]
```

Comment:
Probably #5697 would be merged before this one, so I think it would be worthy to adjust this call to changes introduced it that PR.